packages/target-chrome-core/src/create-chrome-target.js

```diff
@@ -17,6 +17,7 @@ const {
 } = require('@loki/core');
 const presets = require('./presets.json');
 
+const RETRY_LOADING_STORIES_TIMEOUT = 10000;
```

Comment:
This value seems a bit high, how did you come up with it?

Reply:
I'll merge it as is now and we can follow up with tweaks if needed.

Reply:
I would’ve taken a lower value, but I would only do that if we did something like an exponential back off, start slow and iterate maybe 10 times, doubling the wait every time.